.travis.yml

```
@@ -11,12 +11,14 @@ env:
   - STACK=heroku-18 REDIS_VERSION=3
```

I think we should also test on Heroku-16, given it's still supported.

```suggestion
  - STACK=heroku-16 REDIS_VERSION=6
  - STACK=heroku-18 REDIS_VERSION=3
```